Jenkinsfile

```diff
@@ -33,7 +33,7 @@ def rocmtestnode(Map conf) {
         }
     }
     node(name) {
-        withEnv(['HSA_ENABLE_SDMA=0', 'MIOPEN_DEBUG_GCN_ASM_KERNELS=0']) {
+        withEnv(['HSA_ENABLE_SDMA=0']) {
```

Comment:
Do you require this change still ?

Reply:
I can check to see, but I was getting errors with this env on Jenkins with the python backend tests.